pkg/apis/core/v1/resource_types.go

```diff
@@ -19,13 +23,5 @@ type Resource struct {
 	Attributes json.RawMessage `json:"attributes"`
 }
 
-// anxcloud:object:hooks=ResponseFilterHook,RequestBodyHook,FilterRequestURLHook
-
 // ResourceWithTag is a virtual Object used to add (Create) or remove (Destroy) a tag to/from a Resource.
-type ResourceWithTag struct {
-	// Identifier of the Resource which tags to change
-	Identifier string `anxcloud:"identifier"`
-
-	// Name of the Tag to add or remove from the resource
-	Tag string
-}
+type ResourceWithTag = internal.ResourceWithTag
```

Comment:
Not a big fan of 'aliasing internals'.. But maybe it's okay, since we have `Tag`, `Untag` & `ListTags` helper functions in `corev1`? I couldn't find any other solution to resolve import cycles.